### PR TITLE
New events

### PR DESCRIPTION
**Pull request by iopietro**
`[Exiled.Events]`
+ Added ChangingRadioPreset and UsingRadioBattery events.
+ Some clean up.

### CODE REVIEW

**Review by galaxy119 — APPROVED** [review]
lgtm

File: Exiled.Events/Handlers/Player.cs [hunk 1]
```diff
@@ -292,6 +290,16 @@ public static class Player
         /// </summary>
         public static event CustomEventHandler<ChangingIntercomMuteStatusEventArgs> ChangingIntercomMuteStatus;
 
+        /// <summary>
+        /// Invoked before an user's radio battery charge is changed.
```
Comment (iRebbok) [inline Exiled.Events/Handlers/Player.cs:294]:
```suggestion
        /// Invoked before a user's radio battery charge is changed.
```

File: Exiled.Events/Handlers/Player.cs [hunk 2]
```diff
@@ -292,6 +290,16 @@ public static class Player
...
+        /// </summary>
+        public static event CustomEventHandler<UsingRadioBatteryEventArgs> UsingRadioBattery;
+
+        /// <summary>
+        /// Invoked before an user's radio preset is changed.
```
Comment (iRebbok) [inline Exiled.Events/Handlers/Player.cs:299]:
```suggestion
        /// Invoked before a user's radio preset is changed.
```

File: Exiled.Events/Handlers/Player.cs [hunk 3]
```diff
@@ -609,5 +617,17 @@ public static class Player
         /// </summary>
         /// <param name="ev">The <see cref="ChangingIntercomMuteStatusEventArgs"/> instance.</param>
         public static void OnChangingIntercomMuteStatus(ChangingIntercomMuteStatusEventArgs ev) => ChangingIntercomMuteStatus.InvokeSafely(ev);
+
+        /// <summary>
+        /// Called before an user's radio battery charge is changed.
```
Comment (iRebbok) [inline Exiled.Events/Handlers/Player.cs:622]:
```suggestion
        /// Called before a user's radio battery charge is changed.
```

File: Exiled.Events/Handlers/Player.cs [hunk 4]
```diff
@@ -609,5 +617,17 @@ public static class Player
...
+        /// </summary>
+        /// <param name="ev">The <see cref="UsingRadioBatteryEventArgs"/> instance.</param>
+        public static void OnUsingRadioBattery(UsingRadioBatteryEventArgs ev) => UsingRadioBattery.InvokeSafely(ev);
+
+        /// <summary>
+        /// Called before an user's radio preset is changed.
```
Comment (iRebbok) [inline Exiled.Events/Handlers/Player.cs:628]:
```suggestion
        /// Called before a user's radio preset is changed.
```

**Review by BuildBoy12 — APPROVED** [review]
Looks good to me, only thing that'd be nice would be the aforementioned grammatical changes.

**Pull request merged**